cmd/kyverno/main.go

```diff
@@ -483,6 +492,9 @@ func main() {
 		fmt.Println("failed to setup logger", err)
 		os.Exit(1)
 	}
+
+	webhooks.SetDumpFlags(dumpPayload, dumpPayloadTimer, dumpPayloadOps, dumpPayloadKinds)
```

Comment:
This should go in the constructor, not global vars.

Reply:
right

Reply:
@eddycharly which constructor? i'm not sure how to properly pass the flags into the webhooks pkg

Reply:
They should be new argument in `NewServer` func